src/CMakeLists.txt

```diff
@@ -37,11 +37,20 @@ endif()
 # Expose public API
 target_include_directories(benchmark PUBLIC ${PROJECT_SOURCE_DIR}/include)
 
+# Pick installation dir depending on bitness if not specified
+if(NOT DEFINED CMAKE_INSTALL_LIBDIR)
```

Comment:
Well I think that it's a choice between imperative and imperative imperative ;) Me prefer as least imperative as possible but you're the owner ;)